### PR TITLE
Second batch of changes for transport

### PR DESCRIPTION
**Pull request by martinthomson**
Sections 11 through 20.

This breaks due to relying on changes in kramdown-rfc2629.  I'll work around that requirement if things stick on that end.

### CODE REVIEW

**Comment by martinthomson** [comment]
Note: There were a few non-trivial changes in here.  The final batch of commits can be considered atomic as they are in response to specific queries or issues raised by the RPC.

I didn't make separate PRs because most build on top of other edits and conflict management is not fun.

**Review by nibanks — APPROVED** [review]
LGTM

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -5229,27 +5240,27 @@ This section details the transport parameters defined in this document.
...
+of 0 if the transport parameter is absent, unless otherwise stated.
 
 The following transport parameters are defined:
 
 original_destination_connection_id (0x00):
 
-: The value of the Destination Connection ID field from the first Initial packet
-  sent by the client; see {{cid-auth}}.  This transport parameter is only sent
-  by a server.
+: This parameter is the value of the Destination Connection ID field from the
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md:5249]: Oh, is this to be done only for the first element in a list?

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -5415,14 +5426,14 @@ active_connection_id_limit (0x0e):
 
 initial_source_connection_id (0x0f):
 
-: The value that the endpoint included in the Source Connection ID field of the
-  first Initial packet it sends for the connection; see {{cid-auth}}.
+: This is the value that the endpoint included in the Source Connection ID field
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md:5429]: The use of "This is" here (and below) seems to be arbitrary and doesn't seem uniform, but I could be missing something. What is the logic here?

File: draft-ietf-quic-transport.md [hunk 3]
```diff
@@ -6193,8 +6203,8 @@ PROTOCOL_VIOLATION.
 The Retire Prior To field applies to connection IDs established during
 connection setup and the preferred_address transport parameter; see
 {{retire-cid}}. The Retire Prior To field MUST be less than or equal to the
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md:6205]:
```suggestion
{{retire-cid}}. The value in the Retire Prior To field MUST be less than or equal to the value in the
```

File: draft-ietf-quic-transport.md [hunk 4]
```diff
@@ -6193,8 +6203,8 @@ PROTOCOL_VIOLATION.
...
-Sequence Number field. Receiving a value greater than the Sequence Number MUST
-be treated as a connection error of type FRAME_ENCODING_ERROR.
+Sequence Number field. Receiving a value greater than the value in the Sequence
+Number field MUST be treated as a connection error of type FRAME_ENCODING_ERROR.
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md:6207]:
```suggestion
Sequence Number field. Receiving a value in the Retire Prior To field that is greater than that in the Sequence
Number field MUST be treated as a connection error of type FRAME_ENCODING_ERROR.
```

**Review by janaiyengar — COMMENTED** [review]
Thank you for the work, @martinthomson -- a few comments.

File: rfc9000.md [hunk 5]
```diff
@@ -5364,7 +5375,7 @@ Preferred Address {
   Stateless Reset Token (128),
 }
 ~~~
-{: #fig-preferred-address title="Preferred Address format"}
+{: #fig-preferred-address title="Preferred Address Format"}
 
 active_connection_id_limit (0x0e):
```
Comment (janaiyengar) [inline rfc9000.md:5380]: In the description below: s/"The active connection ID limit is"/"This is"

**Review by janaiyengar — APPROVED** [review]
One nit, but LGTM.

**Pull request merged**